lib/content-services/src/lib/document-list/components/document-list.component.spec.ts

```diff
@@ -72,6 +63,7 @@ import { FileAutoDownloadComponent } from './file-auto-download/file-auto-downlo
 import { DocumentListComponent } from './document-list.component';
 import { CustomResourcesService, DocumentListService } from '../public-api';
 import { CommonModule } from '@angular/common';
+import { UnitTestingUtils } from '../../../../../core';
```

Comment:
Since we're in `content-services` the import should come from `@alfresco/adf-core`

Reply:
done